### PR TITLE
DOCS-10893: Tidy some wording in 3.6 draft release notes

### PR DESCRIPTION
**Pull request by stennie**
This change is [Reviewable](https://reviewable.io/reviews/mongodb/docs/3011)

### CODE REVIEW

File: source/release-notes/3.6.txt [hunk 1]
```diff
@@ -199,23 +199,20 @@ Multi-Element Array Updates
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 MongoDB 3.6 adds the following new positional operators for update
-operations:
+operations on arrays and nested arrays:
 
-- The all positional :update:`$[]` operator can support updating all
-  elements in an array.The all positional :update:`$[]` operator can
-  support updating nested arrays.
+- The all positional :update:`$[]` operator updatings all elements in
```
Comment (kay-kim) [inline source/release-notes/3.6.txt]: minor -- updatings -> updates

Comment (stennie) [reply source/release-notes/3.6.txt]: Oops .. missed that from previous edit. Will correct and push an update to this branch.

Comment (stennie) [reply source/release-notes/3.6.txt]: @kay-kim Updated with #f1c8ece

**Comment by kay-kim** [comment]
merged

**Pull request closed without merging**